src/get_assertion_response.rs

```diff
@@ -57,7 +60,11 @@ fn parse_cbor_authdata(
                 if *member == Extension::HmacSecret(None).to_string() {
                     let v = util::cbor_value_to_vec_u8(val)?;
                     let mut hmac_secret_0 = [0u8; 32];
-                    hmac_secret_0.copy_from_slice(&v[0..32]);
+                    let decrypted_secret = enc_aes256_cbc::decrypt_message(
```

Comment:
Thank you for your wonderful contribution!
I think this PR is good and I will merge it in a while.

I have one question.
Is there a way to verify the `decrypted_secret`?

[The specs](https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-20210615.html#sctn-hmac-secret-extension) say I need `CredRandom` to Verify, but I don't know how to get `CredRandom`.

Please let me know if you know how to get `CredRandom`.

Reply:
You can't access CredRandom since that is the secret the authenticator uses to derive credential specific private/public keys, notice that every time the CredRandom is mentioned in the spec it appears in conjunction with authenticator not platform. The spec also doesn't mention that the secret is authenticated in any form unless you verify the attestation I suppose.